pkg/status/status.go

```diff
@@ -247,15 +247,17 @@ func (s *status) updateStatus(previousStatus *configv1.ClusterOperatorStatus) er
 			}
 		}
 
+		// Log Conditions
+		log.Infof("[status] Attempting to set the ClusterOperator status conditions to:")
+		for _, statusCondition := range s.clusterOperator.Status.Conditions {
+			log.Infof("[status] ConditionType: %v ConditionStatus: ConditionMessage: %v", statusCondition.Type, statusCondition.Status, statusCondition.Message)
```

Comment:
You introduced a bug here that none of us caught :-(. `Infof call needs 2 args but has 3 args` Please fix and also use `go-vet` in VScode.

Reply:
Or I can fix it in my PR

Reply:
Ack - I'll address it now.

Reply:
I already fixed it in my PR. Leave it be.

Reply:
Understood, thanks!